integrations/activewindow.cpp

```diff
@@ -67,6 +67,23 @@ ActiveWindowWatcher::ActiveWindowWatcher(QObject *parent)
     }
 }
 
+ActiveWindowWatcher::~ActiveWindowWatcher(){
+    cleanup();
```

Comment:
This can be simplified to just:

m_kwinIface->call("unloadScript", "kiot_activewindow");

There's no reason to check it's valid, or check it's loaded. Worst case it does nothing.